drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -140,6 +140,13 @@ local device_type_attribute_map = {
     clusters.ColorControl.attributes.CurrentSaturation,
     clusters.ColorControl.attributes.CurrentX,
     clusters.ColorControl.attributes.CurrentY
+  },
+  [GENERIC_SWITCH_ID] = {
```

Comment:
Should this be generic Button ID?

Reply:
The button device type is called "Generic Switch"